projects/plugins/jetpack/extensions/blocks/donations/donations-view/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import {
+	ANNUAL_DONATION,
+	DEFAULT_TAB,
+	MONTHLY_DONATION,
+	ONE_TIME_DONATION,
+} from '../common/constants';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import { DonationsIcon } from '../../../shared/icons';
+
+const name = 'donations-view';
+
+const blockTitles = {
+	[ ONE_TIME_DONATION ]: __( 'One Time Donation View', 'jetpack' ),
+	[ MONTHLY_DONATION ]: __( 'Monthly Donation View', 'jetpack' ),
+	[ ANNUAL_DONATION ]: __( 'Annual Donation View', 'jetpack' ),
+};
+
+const settings = {
+	title: __( 'Donations View', 'jetpack' ),
+	description: __( 'Collect one-time, monthly, or annually recurring donations.', 'jetpack' ),
+	icon: DonationsIcon,
+	__experimentalLabel: ( { type } ) => blockTitles[ type ],
+	category: 'earn',
+	attributes: {
+		type: {
+			type: 'string',
+			default: DEFAULT_TAB,
+		},
+		fallbackLinkUrl: {
+			type: 'string',
+			default: '',
+		},
```

Comment:
I think we don't need it in `DonationsView`, but then again, I can't see where it's used in the code anymore. 🤔

Reply:
If it is needed, it will be needed in DonationsView.